src/Services/EmailService.php

```diff
@@ -81,12 +69,22 @@ public function createStalledJobReport(string $subject, string $message, int $jo
             ->setHTMLTemplate('QueuedJobsStalledJob');
     }
 
+    public function createBrokenJobsReport(string $subject, string $message): ?Email
+    {
+        $email = $this->createReport($subject);
+        if ($email === null) {
+            return null;
+        }
+        return $email
+            ->setData([
+                'Message' => $message,
```

Comment:
Pass in the jobs and site as well, as per suggested change to template.

Reply:
Updated